packages/amplify-function-plugin-interface/src/index.ts

```diff
@@ -80,6 +80,7 @@ export type BuildResult = {
 // Result of packaging a function
 export type PackageResult = {
   packageHash?: string; // undefined if no repacking necessary. Otherwise, it is a hash that uniquiely identifies the package
+  zipFileName?: string; // defined for certain plugins like Java
```

Comment:
The dstFilename in the PackageRequest is the zip file. Do you need a different one?

Reply:
nope, changed it